contribs/drt/src/main/java/org/matsim/contrib/drt/run/DrtConfigGroup.java

```diff
@@ -113,6 +113,27 @@ public static DrtConfigGroup getSingleModeDrtConfig(Config config) {
 	@PositiveOrZero
 	public double maxAbsoluteDetour = Double.POSITIVE_INFINITY;// [s]
 
+	@Parameter
+	@Comment(
+		"Defines the maximum allowed absolute detour based on the unsharedRideTime. A linear combination similar to travel time constrain is used"
+			+ "This is the ratio part")
+	@DecimalMin("1.0")
+	public double maxDetourAlpha = Double.POSITIVE_INFINITY;
+
+	@Parameter
+	@Comment(
+		"Defines the maximum allowed absolute detour based on the unsharedRideTime. A linear combination similar to travel time constrain is used"
+			+ "This is the constant part")
+	@PositiveOrZero
+	public double maxDetourBeta = Double.POSITIVE_INFINITY;// [s]
+
+	@Parameter
+	@Comment(
+		"Defines the maximum delay allowed from the initial scheduled pick up time. Once a estimated pick up time is determined, the DRT optimizer"
```

Comment:
Maybe rewrite the second sentence in this way?
"Once the initial pickup time is offered, he DRT optimizer must not exceed the promised time by the `maxAllowedPickupDelay` amount of time."